providers/vrrp_instance.rb

```diff
@@ -47,7 +47,7 @@ def whyrun_supported?
         'virtual_ip_addresses' => new_resource.virtual_ip_addresses,
         'options' => options
       )
-      notifies :restart, 'service[keepalived]'
+      notifies :reload, 'service[keepalived]'
```

Comment:
I'd like to  change notification `:restart` to `:reload` in order to avoid failover on applying cookbook :pray: